packages/block-library/src/button/block.json

```diff
@@ -85,6 +85,16 @@
 			}
 		},
 		"typography": {
+			"__experimentalSkipSerialization": [
+				"fontSize",
+				"lineHeight",
+				"__experimentalFontFamily",
+				"__experimentalFontWeight",
+				"__experimentalFontStyle",
+				"__experimentalTextTransform",
+				"__experimentalTextDecoration",
+				"__experimentalLetterSpacing"
+			],
```

Comment:
Do you think it's okay remove the `__experimental` prefix here and use the [stable keys](https://github.com/WordPress/gutenberg/pull/63401) @aaronrobertshaw?

Reply:
I did wonder the same thing but given that the stabilization is in a state of flux right now it might be better to hold off on using the stable keys. 

There's a good chance I'll have to update them all as part of the latest approach to block support stabilization. So a few more keys are neither here nor there.